packages/editor/src/components/collab-sidebar/index.js

```diff
@@ -270,6 +244,68 @@ export default function CollabSidebar() {
 		enableComplementaryArea( 'core', 'edit-post/collab-sidebar' );
 	};
 
+	const { threads } = useSelect( ( select ) => {
+		const { getCurrentPostId } = select( editorStore );
+		const _postId = getCurrentPostId();
+		const data = !! _postId
+			? select( coreStore ).getEntityRecords( 'root', 'comment', {
+					post: _postId,
+					type: 'block_comment',
+					status: 'any',
+					per_page: 100,
+			  } )
+			: null;
+
+		return {
+			postId: _postId,
+			threads: data,
+		};
+	}, [] );
```

Comment:
Moving the selector in the parent component re-introduces the performance issue fixed in #66670. We should be careful and avoid introducing similar regressions in the future.

P.S. I'm working on a fix and will create a PR shortly.

Reply:
Sorry, I didn't notice this. We now have two sidebars with separate comment orders, thus I had to transfer it to the parent component.